custom_components/tuya_local/devices/cleverio_ar10_ar20.yaml

```diff
@@ -0,0 +1,166 @@
+name: Cleverio AR10 / AR20
```

Comment:
Ue a generic name like "Panel heater" at the top level in case other devices match (many Tuya devices are whitelabel products that are rebranded as local brands, so the same device can have different names in different markets).  Leave the brand under products, as normally the product id will be changed even if other firmware is not (though there are cases where different brands use the same product id too, but if we find that we will adjust to list both).

Reply:
Will do!